app/src/main/java/org/jellyfin/androidtv/ui/playback/MediaManager.java

```diff
@@ -985,6 +985,34 @@ public void resumeAudio() {
         }
     }
 
+    public void fastForward() {
+        UserSettingPreferences prefs = KoinJavaComponent.<UserSettingPreferences>get(UserSettingPreferences.class);
```

Comment:
Injection can be moved to a class property (with inject instead of get)

Reply:
I'm not very familiar with the code base or what CDI is used - do you happen to have an example?

Reply:
Just like line 100 in the same file, add a property for the UserSettingPreferences and then use that property to read the preference from.

https://github.com/jellyfin/jellyfin-androidtv/blob/3ad59c8a4de1805f618fbd5c72ae4fd827180fdb/app/src/main/java/org/jellyfin/androidtv/ui/playback/MediaManager.java#L100